terraform/stage/providers.tf

```diff
@@ -15,6 +15,5 @@ terraform {
     key     = "terraform.tfstate.stage"
     encrypt = "true"
     region  = "us-gov-west-1"
-    profile = "ct-terraform"
```

Comment:
[chore] If this needs to be removed, then we should update the README for running locally to either not put the credentials in that profile (rather keep that) or add a step to locally set the proper ENV variable to access the credentials where they're stored (I think it's `AWS_PROFILE` but double check that)

Reply:
I do believe it needs to be removed, so I can make those changes.